### PR TITLE
Set open files limit to 65535 for kube-proxy

### PR DESCRIPTION
**Pull request by ydp**
**What this PR does / why we need it**:

current kube-proxy limits is default to system value 1024, we need to tune it to 65536 based on:

1.  Antrea scale test issue: https://bugzilla.eng.vmware.com/show_bug.cgi?id=2696987 (current PR did not fix this issue, but address open files limits problem, as for kube-proxy cannot open NodePort, need reproduce and more investigation)
2. upstream kubernetes experience: https://github.com/kubernetes/kubernetes/pull/12443

current kubelet:

```
# cat /proc/$(pidof kubelet)/limits | grep files
Max open files            1000000              1000000              files
```

current apiserver:
```
# cat /proc/$(pidof kube-apiserver)/limits | grep files
Max open files            65535                65535                files
```

**How can this PR be verified?**

Login into worker vm, execute:

Before this PR:
```
cat /proc/$(pidof kube-proxy)/limits | grep files
Max open files            1024                4096                files
```

After this PR:
```
# cat /proc/$(pidof kube-proxy)/limits | grep files
Max open files            65535                65535                files
```

pks-api pipeline:

https://pks.ci.cf-app.com/teams/pks-networking/pipelines/pks-api-1.10.x-dp 

**Is there any change in kubo-deployment?**

**Is there any change in kubo-ci?**

**Does this affect upgrade, or is there any migration required?**

**Which issue(s) this PR fixes:**

**Release note**:

```release-note

```

### CODE REVIEW

File: jobs/kube-proxy/templates/bin/kube_proxy_ctl.erb [hunk 1]
```diff
@@ -45,6 +45,8 @@ start_kubernetes_proxy() {
 
   sed -i "s|HOSTNAMEOVERRIDE|$(get_hostname_override)|g"  /var/vcap/jobs/kube-proxy/config/config.yml
 
+ ulimit -n 65535
```
Comment (lubronzhan) [inline jobs/kube-proxy/templates/bin/kube_proxy_ctl.erb:48]: Maybe move it to pre-start?

File: .final_builds/jobs/kube-proxy/index.yml [hunk 2]
```diff
@@ -7,6 +7,10 @@ builds:
     version: 15572c38aebf328fbf8f840cadb0e1f0cb95df76faf2c14dfae1816da5038436
     blobstore_id: d57616ac-c783-4bed-47cb-43528fdc7ea5
     sha1: sha256:4b813efa211ea1141e54ed62312c5c1ad8564b1a8f484da0255c0cbce9820e56
+  1ab40523587d11a685f0faa519877d9b76e1a3d4c86d9cc1cc73b378a4d84ba7:
```
Comment (lubronzhan) [inline .final_builds/jobs/kube-proxy/index.yml:10]: Are these changes necessary?

Comment (ydp) [reply .final_builds/jobs/kube-proxy/index.yml:10]: I am not sure, just saw former commits has similar, it's added by pipeline, I can remove them.

File: jobs/kube-proxy/templates/bin/kube_proxy_ctl.erb (hunk 1 above)
Comment (ydp) [reply jobs/kube-proxy/templates/bin/kube_proxy_ctl.erb:48]: @lubronzhan  limit or ulimit will only work in current bash script.

Comment (ydp) [reply jobs/kube-proxy/templates/bin/kube_proxy_ctl.erb:48]: we don't have limit in our env, only ulimit is available

File: .final_builds/jobs/kube-proxy/index.yml (hunk 2 above)
Comment (lubronzhan) [reply .final_builds/jobs/kube-proxy/index.yml:10]: I think it's added when you create a dev release. and dev pipeline will create a dev release. hm. maybe just keep them then

Comment (ydp) [reply .final_builds/jobs/kube-proxy/index.yml]: removed it just now...

File: jobs/kube-proxy/templates/bin/kube_proxy_ctl.erb (hunk 1 above)
Comment (lubronzhan) [reply jobs/kube-proxy/templates/bin/kube_proxy_ctl.erb:48]: ok

**Review by lubronzhan — APPROVED** [review]

**Comment by yuyangbj** [comment]
I am ok with this change
